diff --git a/configure.ac b/configure.ac
index 8258b4cc..d2609ff7 100644
--- a/configure.ac
+++ b/configure.ac
@@ -123,7 +123,7 @@ AS_IF([test "x$with_ncurses" != "xno"],
                          [AC_DEFINE([HAVE_NCURSES], [1], [Define to 1 if you have ncurses.]) with_ncurses=yes],
                          [with_ncurses=no])])
 AS_IF([test "x$with_spirv_tools" = "xyes"],
-      [PKG_CHECK_MODULES([SPIRV_TOOLS], [SPIRV-Tools-shared],
+      [PKG_CHECK_MODULES([SPIRV_TOOLS], [SPIRV-Tools],
       [AC_DEFINE([HAVE_SPIRV_TOOLS], [1], [Define to 1 if you have SPIRV-Tools.])])],
       [with_spirv_tools=no])
 
