diff --git a/include/vkd3d_windows.h b/include/vkd3d_windows.h
index 002ff667..097a4787 100644
--- a/include/vkd3d_windows.h
+++ b/include/vkd3d_windows.h
@@ -186,7 +186,7 @@ typedef struct SECURITY_ATTRIBUTES SECURITY_ATTRIBUTES;
         ((type *)((char *)(address) - offsetof(type, field)))

 # ifdef __x86_64__
-#  define __stdcall __attribute__((ms_abi))
+#  define __stdcall
 # else
 #  if (__GNUC__ > 4) || ((__GNUC__ == 4) && (__GNUC_MINOR__ >= 2)) || defined(__APPLE__)
 #   define __stdcall __attribute__((__stdcall__)) __attribute__((__force_align_arg_pointer__))
